### PR TITLE
Refactor/2462 Deprecate store pckg

### PR DESCRIPTION
**Pull request by filrak**
### Related issues

closes https://github.com/DivanteLtd/vue-storefront/issues/2462

### Short description and why it's useful

It's redundant at this point so it's a simplification. Also we will need to to this anyway so it's better to change it now than when we will have tons of modules

### Upgrade Notes and Changelog

- [ ] No upgrade steps required (100% backward compatibility)
- [x] I've updated the [Upgrade notes](https://github.com/DivanteLtd/vue-storefront/blob/develop/doc/Upgrade%20notes.md) and [Changelog](https://github.com/DivanteLtd/vue-storefront/blob/develop/CHANGELOG.md) on how to port existing VS sites with this new feature

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -61,6 +61,7 @@
     "localforage": "^1.7.2",
     "magento2-rest-client": "github:DivanteLtd/magento2-rest-client",
     "pm2": "^2.10.4",
+    "redis-tag-cache": "^1.2.1",
```
Comment (patzick) [inline package.json]: why do we need this dependency here? I cannot find any reference in PR to that

Comment (filrak) [reply package.json]: no idea, shouldn't be there

File: CHANGELOG.md [hunk 2]
```diff
@@ -17,6 +17,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Null sidebar menu data on static page fixed - @filrak (#2456)
 - Fix cannot edit previous steps in checkout - @filrak (#2457)
 
+
+### Depreciated / Removed
```
Comment (patzick) [inline CHANGELOG.md]:
```suggestion
### Deprecated / Removed
```

File: package.json (hunk 1 above)
Comment (filrak) [inline package.json]:
```suggestion
```

**Comment by filrak** [comment]
@pkarw @patzick please review as it's a blocker for few OTHER PRs for me

**Comment by filrak** [comment]
Yeah i guess this dep was missing. I thought we have it somewhere else but not so ill add it to the pckg json

**Pull request merged**